docker-compose.yml

```diff
@@ -14,15 +20,50 @@ services:
       - ./.env:/app/.env
     networks:
       - app-network
-  postgresdb:
-    image: postgres:13.2
-    command: postgres -c 'max_connections=2000'
+  postgres_primary:
```

Comment:
I think best to keep naming consistent: maybe master -> replica ? Confusing to see primary and common here